OpenXMLSDK.Engine/ReportEngine/DataContext/ContextModel.cs

```diff
@@ -127,8 +127,6 @@ public void ReplaceItem(BarModel element, IFormatProvider formatProvider)
                 element.FontName = ReplaceText(element.FontName, formatProvider);
             if (!string.IsNullOrEmpty(element.FontColor))
                 element.FontColor = ReplaceText(element.FontColor, formatProvider);
-            if (!string.IsNullOrEmpty(element.MajorGridlinesColor))
```

Comment:
Why did you remove this condition ?

Reply:
I remove this code because it's not possible to set this color in the context